src/components/EnterpriseApp/data/enterpriseCurationReducer.js

```diff
@@ -47,6 +52,19 @@ function enterpriseCurationReducer(state, action) {
       return { ...state, enterpriseCuration: action.payload };
     case SET_FETCH_ERROR:
       return { ...state, fetchError: action.payload };
+    case SET_TOAST_TEXT: {
```

Comment:
[curious] do we ever clear the `toastText` in this reducer? should we? (i defer to your judgement on this one for sure!)

Reply:
Pushing for no on this only because within the `ContentHighlights` which contain the logic to render the `toasts` array, its dependency array is also the `enterpriseCuration.toastText`, we run the risk of rendering an empty/null toast.

Reply:
Makes sense to me!